modules/ROOT/pages/provisioning-applehv.adoc

```diff
@@ -0,0 +1,56 @@
+= Provisioning Fedora CoreOS on AppleHV
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on MacOS using https://github.com/crc-org/vfkit[vfkit].
+
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You will also need to build or acquire https://github.com/crc-org/vfkit[vfkit].  Prebuilt binaries are also available from its https://github.com/crc-org/vfkit/releases/tag/v0.6.0[releases] page.  Vfkit, like QEMU, has many options which are outside the scope of this provisioning example.  Please consider reading their https://github.com/crc-org/vfkit/tree/main/doc[documentation].
+
+== Booting a new VM on MacOS
+
+This section shows how to boot a new VM with vfkit. Vfkit is known to work with both Intel and Apple Silicon based Macs.
+
+=== Fetching the AppleHV image
+
+Fetch the latest image suitable for your target stream (or https://fedoraproject.org/coreos/download/[download and verify] it from the web).  Remember to download the approach image based on the architecture of your Mac.  Once downloaded, you will also need to decompress the image.
+
+=== Setting up a new VM
+
+Vfkit is not a stateful virtual machine framework. You simply need to run the vfkit binary to start a virtual machine. The following command line will launch a VM with:
+
+* 2 virtual CPUs
+* 2 GB of memory
+* a network device that will receive a IP address from Vfit
+* a GUI console with keyboard and mouse support
+
+.Launching FCOS with Vfkit
+[source, bash]
+----
+IGNITION_CONFIG="/path/to/example.ign"
+IMAGE="/path/to/image.qcow2"
+
+./vfkit --cpus 2 --memory 2048 \
+  --bootloader efi,variable-store=efi-variable-store,create \
+  --device virtio-blk,path=${IMAGE} \
+  --device virtio-net,nat \
+  --ignition ${IGNITION_CONFIG} \
+  --device virtio-input,keyboard \
+  --device virtio-input,pointing \
+  --device virtio-gpu,width=800,height=600 \
+  --gui
+----
+
+Note: The AppleHV hypervisor does not allow you to see early boot and kernel messages.  While you will see a grub boot menu, you will not see anything until later in the boot.
```

Comment:
Suggestion -> `GRUB`

It's odd to me that a user can see GRUB but no kernel messages. That would indicate to me that it's not technically impossible, just that we haven't found the right settings?

Reply:
My understanding is that it is not exposed by Apple.  Is this correct @cfergeau ?

Reply:
but you would think if GRUB can output to a console (either VGA or serial) the kernel could too.. but I certainly don't know this platform at all and will trust the experts

Reply:
I've filed https://github.com/crc-org/vfkit/issues/270 to document this properly on vfkit's end, as the behaviour varies a lot depending on how you start the VM.

Reply:
Thanks @cfergeau. I opened a PR to try to make the change in the images we ship: https://github.com/coreos/fedora-coreos-config/pull/3552